docs/source/getting_started.rst

```diff
@@ -0,0 +1,270 @@
+.. include:: global.rst
+
+Getting Started
+========
+
+This part of the docs will walk you through the main features of remoulade.
+
+Prerequisites
+------
+
+To follow this guide, you will need to set up a new virtual environment and install Remoulade and Requests_::
+
+  $ pip install -U 'remoulade[rabbitmq]' requests
+
+.. _requests: http://docs.python-requests.org
+
+You will also need to install RabbitMQ_, or use Docker_ to run a RabbitMQ image::
+
+  $ docker run -d --name rabbitmq -p 5672:5672 rabbitmq
+
+.. _Docker: https://docs.docker.com/engine/install/ubuntu/
+
+Actors
```

Comment:
`Project setup` maybe ?
What do you think of something like this as plan:
- Prerequisites
- Project setup
- Adding an Actor
- Using a Group of tasks
- Sequential Piping of tasks
- Next Steps (we will see how we manage this part, if we use another page or just continue like this)